File: Corte1/Java/Ejercicio3/Ejercicio3.java
class Punto {
    int x;
    int y;


    public Punto() {
        this.x = 0;
        this.y = 0;
    }


    public Punto(int x) {
        this.x = x;
        this.y = 0;
    }


    public Punto(int x, int y) {
        this.x = x;
        this.y = y;
    }
}

public class Main {
    public static void main(String[] args) {
        Punto p1 = new Punto();
        Punto p2 = new Punto(5);
        Punto p3 = new Punto(3, 4);

        System.out.println(p1.x + ", " + p1.y);
        System.out.println(p2.x + ", " + p2.y);
        System.out.println(p3.x + ", " + p3.y);
    }
}

File: Corte1/Python/Ejercicio3/Ejercicio3.py
class Punto:
    def __init__(self, x=0, y=0):
        self.x = x
        self.y = y


p1 = Punto()
p2 = Punto(5)
p3 = Punto(3, 4)

print(p1.x, p1.y)
print(p2.x, p2.y)
print(p3.x, p3.y)
